taxi/admin.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Car, Driver, Manufacturer
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    list_filter = ["manufacturer"]
+    search_fields = ["model"]
+
+
+@admin.register(Driver)
+class DriverAdmin(UserAdmin):
+    list_display = UserAdmin.list_display + ("license_number",)
+    fieldsets = UserAdmin.fieldsets + (("Additional info", {"fields": ("license_number",)}),)
+    add_fieldsets = UserAdmin.add_fieldsets + (
+        ("Additional info", {"fields": ("license_number",)}),
+    )
+
+
+admin.site.register(Manufacturer)
```

Comment:
You should use one approach for registering your models, either function-based or with the use of `admin.register` decorator.